images/macos/toolsets/toolset-11.json

```diff
@@ -14,19 +14,26 @@
     "xamarin": {
         "vsmac": "latest",
         "mono-versions": [
-            "6.12.0.162"
+            "6.12.0.174"
         ],
         "ios-versions": [
-            "15.8.0.0", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6", "14.20.0.24", "14.16.0.5", "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2"
+            "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6", "14.20.0.24", "14.16.0.5", "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2"
         ],
         "mac-versions": [
-            "8.8.0.0", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27", "7.10.0.5", "7.8.2.5", "7.4.0.10", "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2"
+            "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27", "7.10.0.5", "7.8.2.5", "7.4.0.10", "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2"
         ],
         "android-versions": [
-            "12.0.0.3", "11.3.0.4", "11.2.2.1", "11.1.0.26", "11.0.2.0"
+            "12.2.4.3", "11.3.0.4", "11.2.2.1", "11.1.0.26", "11.0.2.0"
         ],
-        "bundle-default": "6_12_12",
+        "bundle-default": "6_12_16",
```

Comment:
we don't need to change the bundle as it will be incompatible with the old code version installed on the image